README.md

```diff
@@ -2,31 +2,32 @@ Provides programmatic interfaces for the Dai Stablecoin System core contracts.
 
 Import all DSS abstract contracts
 
-```
+```solidity
 import "dss-interfaces/Interfaces.sol";
 ```
 
 Import multiple contracts
 
-```
-import { VatAbstract, VowAbstract } from "dss-interfaces/Interfaces.sol";
+```solidity
+import { DSAuthorityAbstract, DSAuthAbstract } from "dss-interfaces/dapp/DSAuthorityAbstract.sol";
```

Comment:
Not sure about this change. Everything should still be importable from `Interfaces.sol` which is useful for testing. Maybe keep the original line and drop it below the `Import individual contracts` line unless there's another reason to change this.

Reply:
The rational behind this change was to keep the multi-import option but targeting interface contracts with multiple interfaces, but agree on moving it below and maybe we could consider adding a comment on the `Interfaces.sol` import, (e.g. `useful as dss compendium for testing`).

Reply:
should be done in bc424a2